From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Sun, 5 Jun 2011 14:12:40 -0400
Subject: fix RELEASE

---
 configure/RELEASE | 20 ++++++++------------
 1 file changed, 8 insertions(+), 12 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index 8018180..110c3f6 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -13,30 +13,26 @@ TEMPLATE_TOP=$(EPICS_BASE)/templates/makeBaseApp/top
 # GraphicsMagick is required for NDFileMagick and the URL driver.
 USE_GRAPHICSMAGICK=YES
 
-SUPPORT=/corvette/home/epics/devel
+SUPPORT=$(EPICS_BASE)
 -include $(TOP)/../configure/SUPPORT.$(EPICS_HOST_ARCH)
 
 # ASYN is needed for base classes 
-ASYN=$(SUPPORT)/asyn-4-21
+ASYN=$(EPICS_BASE)
 
 # CALC is needed for scalcout record that SSCAN uses
-CALC=$(SUPPORT)/calc-3-0
+CALC=$(EPICS_BASE)
 
 # BUSY is needed for busy record
-BUSY=$(SUPPORT)/busy-1-4
+BUSY=$(EPICS_BASE)
 
 # SSCAN is needed for scanning threshold on Pilatus and general scanning
-SSCAN=$(SUPPORT)/sscan-2-8-1
+SSCAN=$(EPICS_BASE)
 
 #AUTOSAVE is needed for save_restore
-AUTOSAVE=$(SUPPORT)/autosave-5-0
+AUTOSAVE=$(EPICS_BASE)
 
-AREA_DETECTOR=$(SUPPORT)/areaDetector-1-9-1
+#AREA_DETECTOR=$(PWD)
 
 # EPICS_BASE usually appears last so other apps can override stuff:
-EPICS_BASE=/corvette/usr/local/epics/base-3.14.12.3
+EPICS_BASE=/usr/lib/epics
 -include $(TOP)/../configure/EPICS_BASE.$(EPICS_HOST_ARCH)
-
-#Capfast users may need the following definitions
-#CAPFAST_TEMPLATES=
-#SCH2EDIF_PATH=
