From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Sun, 5 Jun 2011 14:12:40 -0400
Subject: [PATCH] fix RELEASE

---
 configure/RELEASE |   25 +++++++++++--------------
 1 files changed, 11 insertions(+), 14 deletions(-)

diff --git a/configure/RELEASE b/configure/RELEASE
index a7c6be0..d615c87 100644
--- a/configure/RELEASE
+++ b/configure/RELEASE
@@ -8,33 +8,30 @@ TEMPLATE_TOP=$(EPICS_BASE)/templates/makeBaseApp/top
 # define INSTALL_LOCATION_APP here
 #INSTALL_LOCATION_APP=<fullpathname>
 
-SUPPORT=/corvette/home/epics/devel
--include $(TOP)/../configure/SUPPORT.$(EPICS_HOST_ARCH)
+GLIBPREFIX=/usr
+
+SUPPORT=$(EPICS_BASE)
 
 # ASYN is needed for sockets and for smartControl
-ASYN=$(SUPPORT)/asyn-4-17
+ASYN=$(EPICS_BASE)
 
 # CALC is needed for scalcout record that SSCAN uses
-CALC=$(SUPPORT)/calc-2-8
+CALC=$(EPICS_BASE)
 
 # BUSY is needed for busy record
-BUSY=$(SUPPORT)/busy-1-3
+BUSY=$(EPICS_BASE)
 
 # SSCAN is needed for scanning threshold on Pilatus
-SSCAN=$(SUPPORT)/sscan-2-6-6
+SSCAN=$(EPICS_BASE)
 
 # MCA is needed for fast accumulation of ROI counts
-MCA=$(SUPPORT)/mca-7-0
+MCA=$(EPICS_BASE)
 
 #AUTOSAVE is needed for save_restore
-AUTOSAVE=$(SUPPORT)/autosave-4-7
+AUTOSAVE=$(EPICS_BASE)
 
-AREA_DETECTOR=$(SUPPORT)/areaDetector-1-7
+#AREA_DETECTOR=$(PWD)
 
 # EPICS_BASE usually appears last so other apps can override stuff:
-EPICS_BASE=/corvette/usr/local/epics/base-3.14.12.1
--include $(TOP)/../configure/EPICS_BASE.$(EPICS_HOST_ARCH)
+EPICS_BASE=/usr/lib/epics
 
-#Capfast users may need the following definitions
-#CAPFAST_TEMPLATES=
-#SCH2EDIF_PATH=
-- 
