From: Michael Davidsaver <mdavidsaver@bnl.gov>
Date: Mon, 9 Apr 2012 15:52:51 -0400
Subject: include system.dbd in all IOCs

---
 ADApp/ADSrc/commonDriverSupport.dbd | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ADApp/ADSrc/commonDriverSupport.dbd b/ADApp/ADSrc/commonDriverSupport.dbd
index 54610ed..9118f58 100644
--- a/ADApp/ADSrc/commonDriverSupport.dbd
+++ b/ADApp/ADSrc/commonDriverSupport.dbd
@@ -1,4 +1,5 @@
 include "base.dbd"
+include "system.dbd"
 include "ADSupport.dbd"
 include "NDPluginSupport.dbd"
 include "calcSupport.dbd"
