Index: areadetector-1.7+dfsg1-6/ADApp/ADSrc/Makefile
===================================================================
--- areadetector-1.7+dfsg1-6.orig/ADApp/ADSrc/Makefile	2011-11-03 15:24:09.000000000 +0100
+++ areadetector-1.7+dfsg1-6/ADApp/ADSrc/Makefile	2014-01-31 23:02:12.421096963 +0100
@@ -24,8 +24,10 @@
 INC += paramAttribute.h
 INC += asynNDArrayDriver.h
 INC += ADDriver.h
-INC += tinyxml.h
-INC += tinystr.h
+#INC += tinyxml.h
+#INC += tinystr.h
+
+USR_CXXFLAGS += -DTIXML_USE_STL
 
 LIBRARY_IOC = ADBase
 LIB_SRCS += NDArray.cpp
@@ -33,7 +35,7 @@
 LIB_SRCS += ADDriver.cpp
 LIB_SRCS += PVAttribute.cpp
 LIB_SRCS += paramAttribute.cpp
-LIB_SRCS += tinyxml.cpp tinystr.cpp tinyxmlparser.cpp tinyxmlerror.cpp
+#LIB_SRCS += tinyxml.cpp tinystr.cpp tinyxmlparser.cpp tinyxmlerror.cpp
 LIB_SRCS += parseAreaPrefixes.c
 
 LIB_LIBS += asyn
Index: areadetector-1.7+dfsg1-6/ADApp/commonDriverMakefile
===================================================================
--- areadetector-1.7+dfsg1-6.orig/ADApp/commonDriverMakefile	2014-01-31 19:50:50.126883375 +0100
+++ areadetector-1.7+dfsg1-6/ADApp/commonDriverMakefile	2014-01-31 22:35:43.742108891 +0100
@@ -64,8 +64,8 @@
 #LIB_SYS_LIBS_Linux     += tiff jpeg z
 #PROD_SYS_LIBS_Linux    += tiff jpeg z
 
-LIB_SYS_LIBS_Linux      += gomp xml2 png12 bz2 
-PROD_SYS_LIBS_Linux     += gomp xml2 png12 bz2
+LIB_SYS_LIBS_Linux      += gomp xml2 png12 bz2 tinyxml
+PROD_SYS_LIBS_Linux     += gomp xml2 png12 bz2 tinyxml
 
 LIB_LIBS_solaris         += NeXus hdf5 jpeg z
 PROD_LIBS_solaris        += NeXus hdf5 jpeg z
Index: areadetector-1.7+dfsg1-6/ADApp/ADSrc/asynNDArrayDriver.cpp
===================================================================
--- areadetector-1.7+dfsg1-6.orig/ADApp/ADSrc/asynNDArrayDriver.cpp	2011-11-03 15:24:09.000000000 +0100
+++ areadetector-1.7+dfsg1-6/ADApp/ADSrc/asynNDArrayDriver.cpp	2014-01-31 22:45:18.738049111 +0100
@@ -19,7 +19,7 @@
 #include <epicsMutex.h>
 #include <cantProceed.h>
 
-#include "tinyxml.h"
+#include <tinyxml.h>
 #include "PVAttribute.h"
 #include "paramAttribute.h"
 #include <asynDriver.h>
Index: areadetector-1.7+dfsg1-6/ADApp/adscSrc/Makefile
===================================================================
--- areadetector-1.7+dfsg1-6.orig/ADApp/adscSrc/Makefile	2014-01-31 23:09:01.435931401 +0100
+++ areadetector-1.7+dfsg1-6/ADApp/adscSrc/Makefile	2014-01-31 23:09:22.564080293 +0100
@@ -118,6 +118,7 @@
 #PROD_LIBS_Linux        += tiff jpeg z
 # To use system versions of these libraries use this line
 #PROD_SYS_LIBS_Linux    += tiff jpeg z
+PROD_SYS_LIBS_Linux    += tinyxml
 
 #PROD_SYS_LIBS_Darwin  += NeXus mfhdf df hdf5 mxml tiff jpeg z
 
