
Index: areadetector/ADApp/ADSrc/parseAreaPrefixes.c
===================================================================
--- areadetector.orig/ADApp/ADSrc/parseAreaPrefixes.c	2014-02-12 12:20:06.200312820 +0100
+++ areadetector/ADApp/ADSrc/parseAreaPrefixes.c	2014-02-12 12:20:06.196312793 +0100
@@ -41,7 +41,7 @@
 	while (!status) {
 		status=dbFirstRecord(pdbentry);
 		while (!status) {
-			sprintf(inBuffer, dbGetRecordName(pdbentry));
+			sprintf(inBuffer, "%s", dbGetRecordName(pdbentry));
 			status = dbFindInfo(pdbentry, "ADType");
 			if (!status) {
 				if (nFound != 0){
Index: areadetector/ADApp/pluginSrc/NDFileHDF5.cpp
===================================================================
--- areadetector.orig/ADApp/pluginSrc/NDFileHDF5.cpp	2014-02-12 12:20:06.084312059 +0100
+++ areadetector/ADApp/pluginSrc/NDFileHDF5.cpp	2014-02-12 12:24:45.950170267 +0100
@@ -413,13 +413,13 @@
   if (storeAttributes == 1)  this->closeAttributeDataset();
   if (storePerformance == 1) this->writePerformanceDataset();
 
-  asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF dataspace %p\n", driverName, functionName, this->dataspace);
+  asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF dataspace %p\n", driverName, functionName, (void*) this->dataspace);
   H5Sclose(this->dataspace);
-  //asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF filespace %p\n", driverName, functionName, this->file);
+  //asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF filespace %p\n", driverName, functionName, (void*) this->file);
   //H5Sclose(this->filespace);
-  asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF cparms %p\n", driverName, functionName, this->cparms);
+  asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF cparms %p\n", driverName, functionName, (void*) this->cparms);
   H5Pclose(this->cparms);
-  asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF datatype %p\n", driverName, functionName, this->datatype);
+  asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing HDF datatype %p\n", driverName, functionName, (void*) this->datatype);
   H5Tclose(this->datatype);
 
   asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s closing groups\n", driverName, functionName);
@@ -1192,7 +1192,7 @@
       this->virtualdims[i] = numCapture;
       //asynPrint(this->pasynUserSelf, ASYN_TRACE_FLOW, "%s::%s extradim=%d ncapture=%d\n",
       //        driverName, functionName, i, numCapture);
-      epicsSnprintf(this->ptrDimensionNames[i], DIMNAMESIZE, extradimdefs[MAXEXTRADIMS - extradims +i].dimName);
+      epicsSnprintf(this->ptrDimensionNames[i], DIMNAMESIZE, "%s", extradimdefs[MAXEXTRADIMS - extradims +i].dimName);
     }
   }
 
